tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	parser "github.com/kubevirt/monitoring/pkg/metrics/parser"
+	"kubevirt.io/ssp-operator/internal/operands/metrics"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+// excludedMetrics defines the metrics to ignore.
+// open bug: https://bugzilla.redhat.com/show_bug.cgi?id=2219763
+// Do not add metrics to this list!
+var excludedMetrics = map[string]struct{}{
```

Comment:
This is fine for now but what's stopping us from hosting this logic outside of the repo and just passing these metric names in as arguments?

Reply:
We send the metrics list as an argument after ignoring this metrics (this are the metrics that failed the linter). 
In the near future after the names will be fixed this list will be removed.

Reply:
`tools/prom-metrics-collector` is about collecting SSP metrics and providing them to the linter (with a list of excluded ones), so not sure what alternative you're suggesting here.

Reply:
External tooling outside of our tree that given the correct arguments could build this list of metrics for the project. Again, supernit this works for now.

Reply:
NOt sur if I get it right, but I don't want specific metrics names to appears in the general linter code.